website_sale_google_shopping/static/description/index.html

```diff
@@ -0,0 +1,210 @@
+<p><img src="https://img.shields.io/badge/licence-AGPL--3-blue.svg"></p>
```

Comment:
Seems like this duplicates a lot of info provided in the ReadMe. I'm not sure of the best practice of what goes where TBH, but it seems to me that we should not duplicate everything.